lib/vagrant-openshift/resources/configure_docker.sh

```diff
@@ -23,24 +23,26 @@ elif lvdisplay centos >/dev/null 2>&1; then
 fi
 
 if [[ -n "${VG}" ]]; then
-    lvcreate -n docker-data -l 70%FREE /dev/${VG}
-    lvcreate -n docker-metadata -l 17%FREE /dev/${VG}
-    lvcreate -n openshift-xfs-vol-dir -l 100%FREE /dev/${VG}
+    lvcreate -n openshift-xfs-vol-dir -l 30%FREE /dev/${VG}
     mkfs.xfs /dev/${VG}/openshift-xfs-vol-dir
     mkdir -p /mnt/openshift-xfs-vol-dir
     echo /dev/${VG}/openshift-xfs-vol-dir /mnt/openshift-xfs-vol-dir xfs gquota 1 1 >> /etc/fstab
     mount /mnt/openshift-xfs-vol-dir
     chown -R "${SSH_USER}:${SSH_USER}" /mnt/openshift-xfs-vol-dir
 
-    DOCKER_STORAGE_OPTIONS="  -s devicemapper"
-    DOCKER_STORAGE_OPTIONS+=" --storage-opt dm.datadev=/dev/${VG}/docker-data"
-    DOCKER_STORAGE_OPTIONS+=" --storage-opt dm.metadatadev=/dev/${VG}/docker-metadata"
+    DOCKER_STORAGE_OPTIONS="-s devicemapper"
```

Comment:
Yeah, looks like docker-storage-setup actually fails when there's something already in /etc/sysconfig/docker-storage:
```
Running ssh/sudo command 'SSH_USER='ec2-user' /home/ec2-user//resources/configure_docker.sh' with timeout 1800. Attempt #0
  Logical volume "openshift-xfs-vol-dir" created.
meta-data=/dev/docker-vg/openshift-xfs-vol-dir isize=512    agcount=4, agsize=392960 blks
         =                       sectsz=512   attr=2, projid32bit=1
         =                       crc=1        finobt=0, sparse=0
data     =                       bsize=4096   blocks=1571840, imaxpct=25
         =                       sunit=0      swidth=0 blks
naming   =version 2              bsize=4096   ascii-ci=0 ftype=1
log      =internal log           bsize=4096   blocks=2560, version=2
         =                       sectsz=512   sunit=0 blks, lazy-count=1
realtime =none                   extsz=4096   blocks=0, rtextents=0
INFO: Volume group backing root filesystem could not be determined
ERROR: Storage is already configured with d driver. Can't configure it with devicemapper driver. To override, remove /etc/sysconfig/docker-storage and retry.
```

If I delete /etc/sysconfig/docker-storage and re-run docker-storage-setup, it succeeds.